docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>In the Projects page &gt; <a href="../monitoring/model-monitoring-deployment.html#model-endpoint-summary-list" target="_blank">Model endpoint summary list</a>, you can see the new two model endpoints, including their drift status:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>